packages/@aws-cdk/aws-iam/test/role.test.ts

```diff
@@ -235,6 +235,49 @@ describe('IAM role', () => {
     });
   });
 
+  describe('role path', () => {
+    test('can be used to specify the path', () => {
+      const stack = new Stack();
+
+      new Role(stack, 'MyRole', { path: '/', assumedBy: new ServicePrincipal('sns.amazonaws.com') });
+
+      expect(stack).toHaveResource('AWS::IAM::Role', {
+        Path: '/',
+      });
+    });
+
+    test('must be between 1 chars and 512 chars', () => {
```

Comment:
This may just be my preference here, but I'd prefer to only see one test case per test.

Reply:
Do you mean 1 expect statement in a test? Fixed in ea3a32439144a1f3597dd0bf14e41f3fe16973b9